.gitignore

```diff
@@ -4,3 +4,4 @@ run/running.ini
 .DS_Store
 run/protos/*_pb2.py
 bq-schema
+generated
```

Comment:
nit: can you config your editor to include a newline at the end of files?

Reply:
Done.